### DISCUSSION TITLE
Show app icon when there isnt any large image displaying

### DISCUSSION THREAD

**Discussion by raider3303** [maximmax42/Discord-CustomRP#207] [Ideas]
The title says it all. When there isnt any large image selected, the app icon would show up 
![image](https://github.com/maximmax42/Discord-CustomRP/assets/108889254/96cb795d-59ad-482f-bd64-0ad0aa1aff03)

**Comment by maximmax42** [comment]
I think it's a Discord's limitation. For some reason it doesn't want to show any images when you don't set anything in the presence, even if you have it set in the dev portal.
